core/core_isolate.rs

```diff
@@ -546,13 +539,14 @@ where
 fn serialize_result(
   promise_id: Option<u64>,
   result: Result<Value, ErrBox>,
+  get_error_class_fn: GetErrorClassFn,
 ) -> Buf {
   let value = match result {
     Ok(v) => json!({ "ok": v, "promiseId": promise_id }),
     Err(err) => json!({
       "promiseId": promise_id ,
       "err": {
-        "kind": "Error",
+        "class": (get_error_class_fn)(&err),
```

Comment:
I'm also in favor of keeping the name "kind" - it becomes a class only in JS world

Reply:
Noted... but the class has no purpose outside of the JS world so why would we have a specific term for it?